cmd/tcpgoon.go

```diff
@@ -0,0 +1,24 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/dachad/tcpgoon/cmdutil"
+	"github.com/dachad/tcpgoon/debugging"
+	"github.com/dachad/tcpgoon/mtcpclient"
+	"github.com/dachad/tcpgoon/tcpclient"
+)
+
+func run(params tcpgoonParams) {
```

Comment:
if we ever have more than one command... whats your idea about the structure? I've been checking the skeleton https://github.com/tcnksm/gcli generates, and its quite nice (but does not support cobra). Maybe you want to take a look and see if this affects to the existence/name of this tcpgoon.go

Reply:
My idea would be create new files for each subcommand and call them from the main.go, something like https://github.com/gohugoio/hugo/blob/master/commands/hugo.go#L185
and implement them like this https://github.com/gohugoio/hugo/blob/master/commands/server.go. 
So, to make it more clearer I could rename root.go to tcpgoon.go and consolidate this root command...

Reply:
:clap: